src/pages/SchoolsGridPage.tsx

```diff
@@ -72,6 +72,14 @@ const LoadingCard = styled(Card)`
   }
 `;
 
+interface ResultsProps {
+  schools: any;
+  error: any;
+  page: number;
+  count: number;
+  onPageChange: (page: number) => void;
+}
+
 const Results: FC<any> = ({ schools, error, page, count, onPageChange }) => {
```

Comment:
Why don't we use above interface here?

Reply:
my mistake - I started to write this interface, but it needs more time and types of 'schools', so I wanted to move this to the moment when we will have type of 'schools' data